pjsip-apps/src/samples/simpleua.c

```diff
@@ -896,18 +896,18 @@ static void call_on_media_update( pjsip_inv_session *inv,
     }
 
     /* Start the UDP media transport */
-    pjmedia_transport_media_start(g_med_transport[0], 0, 0, 0, 0);
+    status = pjmedia_transport_media_start(g_med_transport[0], 0, 0, 0, 0);
```

Comment:
add `pj_assert(status == PJ_SUCCESS)` here, or `if (status != PJ_SUCCESS) {print_some_log; return;}`

Otherwise the returned status will be useless. May as well remove it (back to the original version), which is actually okay since as mentioned previously, these sample apps are not meant to have the most comprehensive error checking.

Reply:
Hi @sauwming, thanks for pointing out. Please see my updated commit.